compiler/plugins/target/AMD-AIE/iree-amd-aie/Target/AIETarget.cpp

```diff
@@ -38,6 +38,43 @@ static llvm::cl::opt<std::string> clEnableAMDAIEUkernels(
                    "unprefixed microkernels to enable, e.g. `matmul`."),
     llvm::cl::init("none"));
 
+// Utility to find aie.device Op corresponding to the export Op, We have
```

Comment:
Nit: Utility to find aie.device Op corresponding to the export Op. For example, we have